lib/baw-workers/partial_payload.rb

```diff
@@ -0,0 +1,151 @@
+module BawWorkers
+
+  class PartialPayloadMissing < StandardError;
+  end
+
+  class InconsistentBasePayload < StandardError;
+  end
+
+  # Stores Redis string payloads in multiple parts.
+  # Useful for splitting apart large payloads that are stuck in, for example, Redis LISTs.
+  # In the common scenario of using a LIST to store job payloads for a processing system, the large invariant parts
+  # of payloads can be split apart and stored separately. We experience 92-96% less memory usage using this technique
+  # on very large payloads for large job sets (e.g. 1000 or more).
+  # This technique should be used when you have at least 100 payloads which are largely identical.
+  class PartialPayload
+    @communicator = BawWorkers::Config.redis_communicator
+
+    class << self
+      PARTIAL_PAYLOAD_KEY = "payload_base"
```

Comment:
Yeah, they don't.  The constant name describes what the value is used for, the value makes more sense when you're looking at a payload ('why is this payload so small? oh I see it has a base payload over there...'). I thought this was self explanatory?